firefox-ios/Client/TabManagement/Tab.swift

```diff
@@ -441,6 +436,7 @@ class Tab: NSObject, ThemeApplicable, FeatureFlaggable, ShareTab {
     }
 
     var profile: Profile
+    var zoomPageManager: ZoomPageManager
```

Comment:
I'm not sure if this is the best place or `ZoomPageManager` should be one for the app and moved to BVC or could be even inverse the `ZoomPageManager` could hold the `Tab` and update the pageZoom and BVC should only have a reference to the ZoomBar to hide/show